modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -0,0 +1,422 @@
+        nativeInputValue.set.call(this.inputRef.current, newValue);
+      }
+
+      let event: Event;
+      if (typeof Event === 'function') {
+        // modern browsers
+        event = new Event('input', {bubbles: true});
+      } else {
+        // IE 11
+        event = document.createEvent('Event');
+        event.initEvent('input', true);
+      }
+
+      this.inputRef.current.dispatchEvent(event);
+    }
+  };
+
+  getTextFromElement = (children?: React.ReactNode) => {
+    let text = '';
+    React.Children.map(children, child => {
+      if (child == null || typeof child === 'boolean' || child === {}) {
+        text += '';
+      } else if (typeof child === 'string' || typeof child === 'number') {
+        text += child.toString();
+      } else if (Array.isArray(child)) {
+        text += this.getTextFromElement(child);
+      } else if ('props' in child) {
+        text += this.getTextFromElement(child.props.children);
+      }
+    });
+    return text;
+  };
+
+  buildStatusString = (listCount: number): string => {
+    return `There ${listCount === 1 ? 'is' : 'are'} ${listCount} suggestion${
+      listCount === 1 ? '' : 's'
+    }.`;
+  };
+
+  handleAutocompleteClick = (event: React.SyntheticEvent, menuItemProps: MenuItemProps): void => {
+    if (menuItemProps.isDisabled) {
+      return;
+    }
+    this.setState({showingAutocomplete: false, isFocused: false});
+    this.setInputValue(this.getTextFromElement(menuItemProps.children));
+    if (menuItemProps.onClick) {
+      menuItemProps.onClick(event);
+    }
+  };
+
+  focusInput = () => {
+    if (this.inputRef.current) {
+      this.inputRef.current.focus();
+    }
+  };
+
+  handleFocus = (event: React.FocusEvent) => {
+    this.setState({isFocused: true});
+
+    if (this.props.onFocus) {
+      this.props.onFocus(event);
+    }
+  };
+
+  handleBlur = (event: React.FocusEvent) => {
+    if (this.comboboxRef.current) {
+      let target: EventTarget | null = event.relatedTarget;
+      if (target === null) {
+        // IE11 swaps related and active target before it fires the blur event
+        target = document.activeElement;
+      }
+      if (target && this.comboboxRef.current.contains(target as Element)) {
+        return;
+      }
+    }
+
+    this.setState({isFocused: false});
+
+    if (this.props.onBlur) {
+      this.props.onBlur(event);
+    }
+  };
+
+  resetSearchInput = (): void => {
+    this.setInputValue('');
+    this.focusInput();
+  };
+
+  handleKeyboardShortcuts = (event: React.KeyboardEvent): void => {
+    if (event.ctrlKey || event.altKey || event.metaKey || !this.props.autocompleteItems) {
+      return;
+    }
+    const currentIndex = this.state.selectedAutocompleteIndex;
+    const autoCompleteItemCount = this.props.autocompleteItems.length;
+    const firstItem = 0;
+    const lastItem = autoCompleteItemCount - 1;
+    let nextIndex = null;
+
+    switch (event.key) {
+      case 'ArrowUp':
```

Comment:
These strings are not supported in IE11. See here for IE11 mappings: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Example

Reply:
Dang Ⓜ️💲. Fixed!